src/pat/inject/inject.js

```diff
@@ -37,7 +37,7 @@ define([
 
     var _ = {
         name: "inject",
-        trigger: "a.pat-inject, form.pat-inject, .pat-subform.pat-inject",
+        trigger: ".ui-button.pat-inject, a.pat-inject, form.pat-inject, .pat-subform.pat-inject",
         init: function inject_init($el, opts) {
```

Comment:
The use of `.ui-button` here bothers me. AFAIK this is not a Patternslib class and it's not documented anywhere either. There's nothing intuitive about it.

What kind of markup are you trying to support here?